CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 1.6.1
+
+## Yarn support
+
+Dependencies can now be managed using [Yarn](https://yarnpkg.com)
```

Comment:
it doesn't really matter what version it goes in as it doesn't effect any release, can we just merge it into master without any release notes?

If we want release notes, I'd recommend v2.0.0